packages/components/toggle-button/src/toggle-button.ts

```diff
@@ -11,23 +13,31 @@ declare global {
   }
 }
 
-export type ToggleButtonSize = 'sm' | 'md' | 'lg';
 export type ToggleButtonFill = 'ghost' | 'outline';
+export type ToggleButtonSize = 'md' | 'lg';
 
 /**
- * Let the user toggle between two states.
+ * Let's the user toggle between two states.
  *
  * ```html
  *  <sl-toggle-button>
- *    <sl-icon name="far-gear"></sl-icon>
-      <sl-icon name="fas-gear" slot="pressed"></sl-icon>
-    </sl-toggle-button>
+ *    <sl-icon name="far-gear" slot="default"></sl-icon>
+ *    <sl-icon name="fas-gear" slot="pressed"></sl-icon>
+ *  </sl-toggle-button>
  * ```
  *
  * @slot default - The icon shown in the default state of the button
  * @slot pressed - The icon shown in the pressed state of the button
  */
-export class ToggleButton extends LitElement {
+export class ToggleButton extends ScopedElementsMixin(LitElement) {
```

Comment:
It looks like `toggle-button` and `toggle-group` is missing in the `tsconfig.all.json` file?